include/diaspora.php

```diff
@@ -932,18 +932,20 @@ private static function post_allow($importer, $contact, $is_comment = false) {
 			logger("defining user ".$contact["nick"]." as friend");
 		}
 
-		// We don't seem to like that person
+		// Is this contact wanted?
```

Comment:
Yes, I rewrote this text a bit. I prefer neutral/descriptive text over "we do this here" and such. :-)